src/plugins/prefs/README.md

```diff
@@ -0,0 +1,107 @@
+- infos = Information about the prefs plugin is in keys below
+- infos/author = Name <name@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = maintained reviewed conformant compatible coverage specific tested nodep libc preview experimental difficult unfinished nodoc concept 
+- infos/metadata =
+- infos/description =
+
+## Basics ##
+
+### Preference Types ###
+
+- Default preferences: below `mountpoint/preferences/pref/`.
+- User preferences: below `mountpoint/preferences/user/`.
+- Lock preferences: below `mountpoint/preferences/lock/`.
+
+User preferences will override default preferences. Lock preferences can't be overwritten by user or default preferences. 
+Only user preferences will be saved to `prefs.js`.
+
+### Data Types ###
+
+- `integer`
+- `string`
+- `boolean`
+
+
+## Example ##
+
+Append
+```
+user_pref("elektra.config.file", "/tmp/myPrefs.js");
+user_pref("elektra.config.reload_trigger_port", 12345);
+```
+to `prefs.js` in `$HOME/.mozilla/firefox/<profile_dir>/`
+
+Move `autoconfig/firefox.cfg` to the directory containing the firefox binary (e.g. `/usr/lib/firefox-esr/`)
+
+Move `autoconfig/autoconfig.js` to the preferences directory inside the folder containing the firefox binary (e.g. `/usr/lib/firefox-esr/defaults/pref/`)
+
+
+```
+% kdb mount autoconfig/preload.ini /preload ini
+% kdb export /preload
+
+[open]
+\/tmp\/myPrefs.js =
+\/tmp\/myPrefs.js/generate = system/prefs/gen
+\/tmp\/myPrefs.js/generate/plugin = prefs
+
+
+% kdb mount autoconfig/prefExample.js system/prefs/gen prefs shell execute/set='echo -n "reload"|nc 127.0.0.1 12345'
+% kdb export system/prefs/gen
+
+[functions]
+[preferences]
+[preferences/lock/a/lock]
+1 = lock1
+2 = lock2
+[preferences/pref/a/default]
+1 = 1
+2 = 2
+[preferences/user/a/user]
+f = false
+t = true
+[variables]
+
+% kdb export system/prefs/gen prefs
+
+lockPref("a.lock.1", "lock1");
+lockPref("a.lock.2", "lock2");
+pref("a.default.1", 1);
+pref("a.default.2", 2);
+user_pref("a.user.f", false);
+user_pref("a.user.t", true);
+```
+
+```
+% LD_PRELOAD=/usr/local/lib/libelektraintercept.so firefox-esr "about:config"
```

Comment:
A script that does the preloading will hide the path (similar to `kdb elektrify-getenv` we should have a `kdb elektrify-open` or `kdb elektrify-filesystem`)